resources/translations/es.json

```diff
@@ -0,0 +1,231 @@
+  "ERROR_PERMISSIONS": "Me faltan permisos!",
+  "ERROR_REQUIRED_ROLE": "No tienes el rol requerido para ingresar a la lista de espera!",
+  "ERROR_RESPONSE": "Respuesta inválida",
+  "ERROR_ROLE": "Rol inválido",
+  "ERROR_SET": "No configurado",
+  "ERROR_TIME": "Se te acabó el tiempo!",
+  "ERROR_UNDO": "Sólo puedes deshacer una artida que ha sido reportada o cancelada",
+  "ERROR_UNKNOWN_DESCRIPTION": "Escribe este código en el canal #error-report en el servidor de 6 Mans para que podamos examinarlo. ", 
+  "ERROR_UNKNOWN_ID": " ID del error: {{error}}"
+  "ERROR_UNKNOWN_NAME": "Algo salió mal!",
+  "ERROR_USER": "Algo salió mal!",
+  "ERROR_USER_DM": "{{user}} no pude enviarte un mensaje privado",
+  "FREE_MEMORY": "Memoria vacía",
+  "GENERIC_CAPTAIN": "Capitán {{number}}",
+  "GENERIC_HOME": "Volver al inicio",
+  "GENERIC_LOSE": "Derrota",
+  "GENERIC_MATCH": "Partida {{id}}",
+  "GENERIC_NEGATIVE": "Cancelar",
+  "GENERIC_POSITIVE": "Confirmar",
+  "GENERIC_REPORT": "Reportar partida",
+  "GENERIC_STOP": "Detener este menú",
+  "GENERIC_TEAM": "Equipo {{number}}",
+  "GENERIC_TIME": "Hora",
+  "GENERIC_WIN": "Victoria",
+  "GENERIC_WINNER": "Ganador",
+  "GETTING_PING": "Obteniendo Ping!",
+  "GUILD_COUNT": "Conteo del servidor",
+  "HELP_COMMAND": "Ayuda",
+  "HELP_COMMAND_INFO": "Para más información acerca del comando escribe `/help command [nombre del comando]`",
+  "HELP_COMMAND_INFO_LONG": "Para más información acerca del comando escribe `/help command [command name]`. \nEscribe `/help all` para ver la lista de comandos.",
+  "HELP_COMMAND_NAME_FIND": "Comando con el nombre `{{name}}` no encontrado",
+  "HELP_COMMAND_NAME_INVALID": "Nombre de comando inválido",
+  "HELP_COMMAND_NAME_TITLE": "Ayuda por {{name}}:",
+  "HELP_DELETE_DATA": "¿Quieres realizar una solicitud para eliminar tu información de usuario?",
+  "HELP_DELETE_DATA_DETAIL": "En el servidor anexado, ve al canal #data-deletion-request. Dentro del canal, escribe /deletedata.",
+  "HELP_DEVELOPER": "Este es un comando para **desarrollador**,
+  "HELP_DEVELOPER_DETAIL": "Sólo los desarrolladores del bot pueden usar este comando.",
+  "HELP_INVITE": "¿Quieres usar el bot en tu servidor?",
+  "HELP_INVITE_LINK": "[Invitación al servidor Private6Mans({{link}})",
+  "HELP_ISSUES": "¿Tienes problema?",
+  "HELP_ISSUES_LONG": "Asegúrate de que el bot tenga permisos de leer y enviar mensajes, enviar links y agregar reacciones. Además, asegúrate de que todos en la sala puedan recibir mensajes privados del bot.",
+  "HELP_LIST_DEVELOPER": "Lista de comandos del desarrollador",
+  "HELP_LIST_MODERATOR": "Lista de comandos del moderador",
+  "HELP_LIST_PUBLIC": "Lista de comandos públicos",
+  "HELP_MODERATOR": "Este es un comando de **moderador**",
+  "HELP_MODERATOR_DETAIL": "Sólo usuarios con el rol de moderador designado por Private 6 Mans (/config modrole), o usuarios con los permisos de creador y administrador del servidor pueden usar este comando.",
```

Comment:
@TheBeardedB this isn't the command for mod role anymore is it? Something for me to fix in my English doc too?

Reply:
it's now /config guild mod-role